index.html

```diff
@@ -214,6 +214,94 @@ <h5 class="program__content--header">CC Exhibition</h5>
+          <div class="col">
+            <div class="speakers__card">
+              <div class="speakers__img">
+                <img
+                  src="img/speaker_01.png"
+                  alt="speaker"
+                  title="speaker"
+                  class="img"
+                />
+              </div>
+              <div class="speakers__card--info">
+                <h4>Yochai Benkler</h4>
+                <p class="about">
+                  Lorem ipsum, dolor sit amet consectetur adipisicing.
+                </p>
+                <hr class="speakers__hr" />
+                <p>
+                  Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quod
+                  vitae repellendus, harum modi voluptatum hic.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="col">
+            <div class="speakers__card">
+              <div class="speakers__img">
+                <img
+                  src="img/speaker_03.png"
+                  alt="speaker"
+                  title="speaker"
+                  class="img"
+                />
+              </div>
+              <div class="speakers__card--info">
+                <h4>Yochai Benkler</h4>
+                <p class="about">
+                  Lorem ipsum, dolor sit amet consectetur adipisicing.
+                </p>
+                <hr class="speakers__hr" />
+                <p>
+                  Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quod
+                  vitae repellendus, harum modi voluptatum hic.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="partners">
+        <div class="partners__header">
+          <h5>Partner</h5>
+          <hr class="secondary__hr" />
+        </div>
+        <ul class="partners__list">
+          <li class="partners__list--item">Mozilla</li>
+          <li class="partners__list--item">Google</li>
+          <li class="partners__list--item">NAVER</li>
+          <li class="partners__list--item">daumkakao</li>
+          <li class="partners__list--item">airbnb</li>
+        </ul>
+      </section>
     </main>
+    <footer class="container footer p-1 m-2 d-flex flex-column">
+      <div class="row justify-content-md-center">
+        <div
+          class="footer__icon col col-md-auto d-flex flex-row align-item-center justify-content-center"
+        >
+          <img src="img/logo.png" alt="logo" title="logo" />
+        </div>
+        <div
+          class="footer__text col col-md-auto d-flex flex-column align-item-center justify-content-center mb-0"
+        >
+          <div class="footer__text">
+            <h6 class="footer__text--header mb-0">&copy;2022 Code School</h6>
+            <p class="footer__text--text mb-0">All Rights Reserved.</p>
+          </div>
+        </div>
+      </div>
+    </footer>
```

Comment:
- There are a couple of issues with the footer that you need to take care of before you will be good to go 😄 
- [ ] On the desktop version of both pages, kindly increase the height of the footer as in the required design. Your footer's height is smaller 👍
- Your design:
![Annotation 2022-05-16 190401](https://user-images.githubusercontent.com/85113567/168656759-85e94227-b9ae-4233-952e-ca7c4e34e487.jpg)
- Required design:
![Annotation 2022-05-16 190347](https://user-images.githubusercontent.com/85113567/168656809-c152371b-e6d6-4037-8e95-a4304aad4fa7.jpg)

- [ ] In the mobile version of the about page, a dark partner's section should be present with a white footer. Kindly rectify this issue too 👍 
- Your design:
![Annotation 2022-05-16 190533](https://user-images.githubusercontent.com/85113567/168656987-daeee23c-2199-4c17-9926-c6f92e0c9710.jpg)
- Required design:
![Annotation 2022-05-16 190518](https://user-images.githubusercontent.com/85113567/168657020-2dc19af4-7e73-49b0-9936-e6c5b5007934.jpg)

Reply:
Done now

thanks